database/migrations/2021_11_03_131308_onboarding.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class Onboarding extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('accounts', function (Blueprint $table) {
+            $table->boolean('is_onboarding')->default(false);
+            $table->json('onboarding')->nullable();
```

Comment:
I believe we need to use text or mediumText as column type for cross DB compatibility for MySQL and MariaDB